openflow15/nx_util.go

```diff
@@ -91,6 +91,10 @@ const (
 	NXTTMC_CLEAR
 )
 
+const (
+	AntreaLargePacket = 0x10002320 // Vendor type to mark that VendorHeader message length is overflow.
```

Comment:
where does the number come from?

Reply:
I feel like we should have a general vendor id for Antrea (like there is one for Nicira, ONF, Intel), but we should not make a specific use-case the vendor id. It's confusing and we may have additional use cases in the future.

Reply:
Do you mean VendorID or Experimenter type? In my latest change, I switch to use ExperimenterType for the large packet, which is using the same field with different value from the general PacketIn2, see https://github.com/antrea-io/libOpenflow/pull/80/files#diff-b5525a0f43ec11209c4ad3c3035904678a43e69a9cad27b224eef4a3858d74b7R1346

Reply:
No, I am using `0x10002320` while Nicira is using `0x00002320`, I have added 1<<28